libs/watt/src/lib/components/modal/+storybook/watt-modal-from-class.component.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnInit } from '@angular/core';
+import { WattButtonComponent } from '../../button/watt-button.component';
+import { WATT_MODAL } from '../watt-modal.component';
+import { WattModalService } from '../watt-modal.service';
+import { WattTextFieldComponent } from '../../text-field/watt-text-field.component';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  standalone: true,
+  selector: 'watt-modal-from-class',
+  imports: [WATT_MODAL, WattTextFieldComponent, WattButtonComponent],
+  template: `
+    <watt-modal #modal [title]="title" closeLabel="Close modal" [loading]="isLoading">
+      <watt-text-field [formControl]="exampleFormControl" label="Username" />
+      <watt-text-field [formControl]="exampleFormControl" label="Password" type="password" />
```

Comment:
nitpick: the same control is used on both input fields.

Reply:
Should be fixed now